fuel-core/src/cli/run.rs

```diff
@@ -145,6 +159,18 @@ impl Command {
             }
         });
 
+        let coinbase_recipient = if let Some(coinbase_recipient) = coinbase_recipient {
+            Address::from_str(coinbase_recipient.as_str()).map_err(|err| anyhow!(err))?
+        } else {
+            let consensus_key = consensus_key
+                .as_ref()
+                .cloned()
+                .unwrap_or_else(|| Secret::new(SecretKeyWrapper::default()));
+
+            let sk = consensus_key.expose_secret().deref();
+            Address::from(*sk.public_key().hash())
```

Comment:
Is that the right way to retrieve `Address` from `SecretKey`?

Reply:
it is, but I should've probably used a helper fn from fuel-tx instead.

Reply:
What helper do you mean?

Reply:
Input::owner(&publick_key)